Test failure on /contact pages

https://gitlab.com/mozmeao/www-config/-/pipelines/188094780/builds

```
FAILED tests/functional/test_contact.py::test_spaces_mobile_navigation
```

```
    @pytest.mark.nondestructive
    def test_spaces_mobile_navigation(base_url, selenium_mobile):
        page = SpacesPage(selenium_mobile, base_url, slug='').open()
        assert page.is_mobile_menu_toggle_displayed
>       page.open_spaces_mobile_menu()
base_url   = 'https://bedrock-dev.gcp.moz.works'
page       = <pages.contact.SpacesPage object at 0x7f397852c110>
selenium_mobile = <selenium.webdriver.remote.webdriver.WebDriver (session="6e463c518b604495bd6cc78b8a2ffc54")>
tests/functional/test_contact.py:45: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/pages/contact.py:56: in open_spaces_mobile_menu
    self.wait.until(lambda s: self.is_nav_displayed)

E selenium.common.exceptions.TimeoutException
```

Possibly a regression from https://github.com/mozilla/bedrock/pull/9377

--- a/bedrock/mozorg/templates/mozorg/contact/contact-base.html
+++ b/bedrock/mozorg/templates/mozorg/contact/contact-base.html
@@ -70,4 +70,8 @@
 {% endblock %}
 
 {% block js %}
+  {# Issue 9322 #}
+  {% if self.body_id() != 'contact-landing' %}
+    {{ js_bundle('contact-spaces') }}
+  {% endif %}
 {% endblock %}

--- a/bedrock/mozorg/templates/mozorg/contact/contact-landing.html
+++ b/bedrock/mozorg/templates/mozorg/contact/contact-landing.html
@@ -5,6 +5,7 @@
 {% extends "mozorg/contact/contact-base.html" %}
 
 {% block page_title %}{{ _('Contacts, Spaces and Communities') }} — {{ _('Contact us') }}{% endblock %}
+{% block body_id %}contact-landing{% endblock %}
 
 {% block tab_navigation %}
   {% with current = 'contact' %}

